src/main/kotlin/blackjack/domain/BlackjackGame.kt

```diff
@@ -0,0 +1,42 @@
+package blackjack.domain
+
+class BlackjackGame(
+    val dealer: Dealer = Dealer(),
+    val participants: Participants
+) {
+    fun betParticipant(getBettingMoney: (Participant) -> Int) {
+        participants.forEach { participant ->
+            participant.bettingMoney = getBettingMoney(participant)
+        }
+    }
```

Comment:
[이전 코멘트](https://github.com/next-step/kotlin-blackjack/pull/827#discussion_r1910453099)의 내용은 현재 participant의 setter를 이용해서 직접 participant의 상태 값을 변경하고있는 행위 대신에  participant에게 [메시지를 보내](https://tecoble.techcourse.co.kr/post/2020-04-28-ask-instead-of-getter/) participant가 자신의 상태 값을 자기 스스로 바꾸도록해보면 어떨까하는 이야기였어요! 😃

```suggestion
    fun betParticipant(getBettingMoney: (Participant) -> Int) {
        participants.forEach { participant ->
            participant.bet(getBettingMoney(participant))
        }
    }
```

Reply:
아하 제가 이 두개를 같은 형태로 생각하고 있었는데, 또 다르게 생각해보니 다르게 보이는군요! 감사합니다 ㅎㅎ

Reply:
8f2ef11a7701d3d43b1ec37c367729f58cde5b22